flink-runtime/src/test/java/org/apache/flink/runtime/zookeeper/ZooKeeperStateHandleStoreTest.java

```diff
@@ -110,62 +108,84 @@ public void testAddAndLock() throws Exception {
 
         // Verify
         // State handle created
-        assertEquals(1, store.getAllAndLock().size());
-        assertEquals(state, store.getAndLock(pathInZooKeeper).retrieveState().getValue());
+        assertThat(store.getAllAndLock().size()).isOne();
+        assertThat(store.getAndLock(pathInZooKeeper).retrieveState().getValue()).isEqualTo(state);
 
         // Path created and is persistent
-        Stat stat = ZOOKEEPER.getClient().checkExists().forPath(pathInZooKeeper);
-        assertNotNull(stat);
-        assertEquals(0, stat.getEphemeralOwner());
-
-        List<String> children = ZOOKEEPER.getClient().getChildren().forPath(pathInZooKeeper);
+        Stat stat =
+                zooKeeperExtension
+                        .getZooKeeperClient(
+                                testingFatalErrorHandlerResource.getTestingFatalErrorHandler())
```

Comment:
having a local variable for the client might make the code more readable.

Reply:
additionally, having a private method `getClient()` might help

Reply:
Added the `getZooKeeperClient` method, it's indeed a good ssuggestion, it can reduce a lot of repeated code.